ldms/man/ldmsd_controller.man

```diff
@@ -514,8 +553,16 @@ The update policy name
 .TP
 .BI [interval " interval"]
 .br
-The update interval in micro-seconds. If this is not
-specified, the previously configured value will be used. Optional.
+The update interval, which is an integer followed by a unit string.
+If no unit string is given, the default unit is microseconds.
+A unit string is one of the followings:
+  us -- microseconds
+  ms -- milliseconds
+  s  -- seconds
+  min -- minutes
```

Comment:
'm'
'h'
'd'
seems more concise.

Reply:
@nichamon Maestro currently supports "m" for minutes as well. It doesn't currently support hours or days, but I can add a quick patch to address this once we land on a final format for minutes/hours/days. I'm in favor of shortening it like @tom95858 suggested, as it lends itself less to typos.

Reply:
@tom95858 @nick-enoent I changed the minute, hour, and day strings to 'm', 'h', and 'd', respectively. Also, the patch supports floats, e.g., '1.5s' as in Maestro.